erpnext/hr/doctype/salary_slip/salary_slip.py

```diff
@@ -272,6 +292,25 @@ def calculate_lwp(self, holidays, working_days):
 				lwp = cint(leave[0][1]) and (lwp + 0.5) or (lwp + 1)
 		return lwp
 
+	def calculate_absents(self):
+		absent_days = 0
+
+		attendances = frappe.db.sql('''SELECT status, count(*) as non_working_days
+			FROM `tabAttendance`
+			WHERE
+				status NOT IN ("Present","On leave")
```

Comment:
"On leave" with leave_type as LWP should also be treated as absent.

Reply:
Done

Reply:
"On leave" with leave_type as LWP?